testcases/kernel/containers/netns/netns_helper.sh

```diff
@@ -110,7 +110,9 @@ tst_check_iproute()
 netns_setup()
 {
 	tst_require_root
-	tst_check_cmds ip
+	tst_check_cmds ip modprobe
+
+	modprobe veth > /dev/null 2>&1
```

Comment:
use the -q option and i think you can drop the output redirection

Reply:
I've done the output redirection since it's the approach used across LTP source code. Not sure why -q isn't used... maybe it isn't in the busybox version? Just speculation at this point, I didn't checked.

Reply:
That may be the case here, I googled random modprobe busybox source and it does not seem to be implemented there. The redirection seems to be safer bet here.

Reply:
*I've done the output redirection since it's the approach used across LTP source code. Not sure why -q isn't used... maybe it isn't in the busybox version?*
@fboudra `busybox modprob` **has** `-q` option, please use it. Not all LTP code is a good example :-(.